fury/ui.py

```diff
@@ -1111,11 +1114,18 @@ def __init__(self, text="Text Block", font_size=18, font_family='Arial',
             Makes text italicised.
         shadow : bool
             Adds text shadow.
+        size : (int, int)
+            Size (width, height) in pixels of the text bounding box.
         """
         super(TextBlock2D, self).__init__(position=position)
+        self.scene = None
+        self.bg = bool(bg_color)
```

Comment:
`self.have_bg` instead of `self.bg`

Reply:
Hello @skoudoro , Currently, the background adjusts itself with the font size of the Text Actor, i.e the background expands if the font size is bigger. Do you want me to prevent that and generate a warning instead?

Reply:
I do not have this behavior. When I increase the font_size, the background does not expands.

Reply:
Did you expand it externally? like `text.font_size = 100` ?

Reply:
Yes

Reply:
Oh yes, I did not add that in `font_size.setter`. Also, what should the behaviour be like if the user passes both `size` and `font_size` parameter?

Reply:
Priority to `size`, font_size should adapt.

Reply:
Okay, for now, I am preventing the user from updating the font size if the text overflows the background.

Reply:
Great! Thanks, I will play with bit and keep you updated

Reply:
It does not work for me, I got the following error:

```python
In [3]: from fury import window, ui
In [4]: te = ui.TextBlock2D(position=(50, 50), size=(100, 100), bg_color=(1,1,0))  
In [5]: s = window.ShowManager()                                                                                                         

In [6]: s.scene.add(te)                                                                                                                  

In [7]: s.start()                                                                                                                        

In [8]: te.color =(0,0,0)                                                                                                                

In [9]: s.start()    
In [10]: te.font_size = 50                                                                                                               
ERROR: In /Users/prabhu/src/git/VTKPythonPackage/standalone-build/VTK-source/Rendering/Core/vtkTextActor.cxx, line 902
vtkOpenGLTextActor (0x7fd5f293e120): No render window available: cannot determine DPI.

ERROR: In /Users/prabhu/src/git/VTKPythonPackage/standalone-build/VTK-source/Rendering/Core/vtkTextActor.cxx, line 902
vtkOpenGLTextActor (0x7fd5f293e120): No render window available: cannot determine DPI.

ERROR: In /Users/prabhu/src/git/VTKPythonPackage/standalone-build/VTK-source/Rendering/Core/vtkTextActor.cxx, line 164
vtkOpenGLTextActor (0x7fd5f293e120): UpdateRectangle failed to do so
```

Does it work for you ?

Reply:
Yeah gives me these errors when I close the window and Interactor names are printed when I click on the text block. Can you try running it as a script?

Reply:
Because of this error, the UI is losing the interactor, this is why we see the message from this line https://github.com/fury-gl/fury/blob/master/fury/interactor.py#L385.

It is strange, we need to understand why today.

Reply:
@skoudoro , The problem seems to occur when you execute multiple `sm.start()` commands. The same thing happens when you do it with the `fury-gl/master branch` as well.

Reply:
Also, the vtk warnings appear when you try to set the font size after closing the window. When the window is closed, no such scene exists. The bounding box constraint check that I added needs the scene to exist to calculate the size of the text actor and therefore throws the following error.

Reply:
Ok, thank you for your feedback. I will go ahead and merge it.

Reply:
Thank you @skoudoro